coreprofile/10/_index.md

```diff
@@ -1,37 +1,66 @@
-
-
-* [Jakarta EE Core Profile 10 Release Record](https://projects.eclipse.org/projects/ee4j.jakartaee-platform/releases/core-profile-10)
+* [Jakarta Core Profile 10 Release Record](https://projects.eclipse.org/projects/ee4j.jakartaee-platform/releases/core-profile-10)
+  * [Jakarta EE Platform 10 Release Plan](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee10/JakartaEE10ReleasePlan)
+* [Jakarta Core Profile 10 Specification Document](./jakarta-coreprofile-spec-10.pdf) (PDF)
+* [Jakarta Core Profile 10 Specification Document](./jakarta-coreprofile-spec-10.html) (HTML)
+* [Jakarta Core Profile 10 Javadoc](./apidocs)
+* [Jakarta Core Profile 10 TCK](https://download.eclipse.org/jakartaee/platform/10/jakarta-jakartaeetck-10.0.0.zip) ([sig](https://download.eclipse.org/jakartaee/platform/10/jakarta-jakartaeetck-10.0.0.zip.sig),[sha](https://download.eclipse.org/jakartaee/platform/10/jakarta-jakartaeetck-10.0.0.zip.sha256),[pub](https://raw.githubusercontent.com/jakartaee/specification-committee/master/jakartaee-spec-committee.pub))
+* Compatible Implementations used for [ratification](https://www.eclipse.org/projects/efsp/?version=1.2#efsp-ratification).
+  * [TBD]()
+  
+# Jakarta EE 10 Schedule
+* [Jakarta EE 10 Schedule](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee10/JakartaEE10#jakarta-ee-10-schedule)
 
+# Compatible Implementations
+* [Jakarta EE 10 Compatible Implementations](https://jakarta.ee/compatibility/#tab-10)
 
 # Ballots
 
 ## Plan Review
 
+The Specification Committee Ballot concluded successfully on 2021-08-19 with the following results.
+
+| Representative                                 | Representative for: |  Vote  |
+|------------------------------------------------|---------------------|--------|
+| Kenji Kazumura                                 | Fujitsu             |   +1   |
+| Dan Bandera, Kevin Sutter                      | IBM                 |   +1   |
+| Ed Bratt, Dmitry Kornilov                      | Oracle              |   +1   |
+| Andrew Pielage, Matt Gill                      | Payara              |   +1   |
+| Scott Stark, Mark Little                       | Red Hat             |   +1   |
+| David Blevins, Jean-Louis Monteiro             | Tomitribe           |   +1   |
+| Ivar Grimstad                                  | EE4J PMC            |   +1   |
+| Marcelo Ancelmo, Martijn Verburg               | Participant Members |   +1   |
+| Werner Keil                                    | Committer Members   |   +1   |
+| Jun Qian                                       | Enterprise Members  |   +1   |
+|                                                | Total               | **10** |
+
+The ballot was run in the [jakarta.ee-spec mailing list](https://www.eclipse.org/lists/jakarta.ee-spec/msg01679.html).
+
+## Release Review
+<!--
+TBD
 The Specification Committee Ballot concluded successfully on 2021-05-19 with the following results.
```

Comment:
This is the result of the "Creation and Plan Review". If the plan part is replaced by the ballot above, the creation review should at least be listed.

Reply:
That section is for a future release and is commented out. The creation/plan review is what is referecned as the https://www.eclipse.org/lists/jakarta.ee-spec/msg01679.html vote, so I corrected the date and updated the title.